test/specs/kefir-observable.js

```diff
@@ -0,0 +1,49 @@
+let {Kefir} = require('../test-helpers.coffee');
+
+describe('Kefir.Observable', function() {
+    describe('observe', function() {
+        let em, count, obs, sub;
```

Comment:
Some of those tests fail currently, although I'm not sure why. I'll look into transpiling the tests in Jasmine.